app/views/contexts/list_context.py

```diff
@@ -9,14 +10,14 @@
 class ListContext(Context):
     def __call__(
         self,
-        summary_definition,
-        for_list,
-        return_to=None,
-        edit_block_id=None,
-        remove_block_id=None,
-        primary_person_edit_block_id=None,
-        for_list_item_ids=None,
-    ):
+        summary_definition: Mapping[str, Any],
+        for_list: str,
+        return_to: Optional[str] = None,
+        edit_block_id: Optional[str] = None,
+        remove_block_id: Optional[str] = None,
+        primary_person_edit_block_id: Optional[str] = None,
+        for_list_item_ids: Optional[str] = None,
```

Comment:
`for_list_item_ids` should be Optional Iterable/Sequence of strings.

Reply:
👍